doc/source/security/encrypting-redo-log.rst

```diff
@@ -0,0 +1,37 @@
+.. _encrypting-redo-log:
+
+================================================================================
+Encrypting the Redo Log
+================================================================================
+
+The Redo log can be encrypted with the :variable: `innodb_redo_log_encrypt` variable. The default value for the variable is ``OFF``. The Redo log uses the tablespace encryption key.
+
+.. variable:: innodb_redo_log_encrypt
+
+    :cli:  ``--innodb-redo-log-encrypt``
+    :dyn: Yes
+    :scope: Global
+    :vartype: Text
+    :default: OFF
+
+Defines the redo log data for encrypted tables. 
```

Comment:
?

Reply:
Unaddressed comment.

Reply:
Determines the encryption for redo log data for tables. The encryption of redo
 20 log data, by default, is 'OFF'.